management_interface/management_interface/models.py

```diff
@@ -65,6 +68,9 @@ class RegisteredManager(BaseModel):
     )
     cqc_registered_manager_id = models.CharField(max_length=128, help_text="1-XXXXXXXX")
 
+    def __str__(self):
+        return f"{self.given_name} {self.family_name} ({self.cqc_registered_manager_id})"
```

Comment:
```suggestion
        return f"{self.family_name}, {self.given_name} ({self.cqc_registered_manager_id})"
```

So that when we sort them on this, they'll come out alphabetical by family name. I think that's what we want.